locales/frFR.lua

```diff
@@ -926,21 +926,21 @@ for key,value in pairs({
 	[34] = "Vieille cruche",	-- Old Jug
 	[35] = "Coffre du capitaine",	-- Captain's Footlocker
 	[36] = "Tonneau cassé",	-- Broken Barrel
-	[47] = "Avis de recherche !",	-- Wanted: Lieutenant Fangore
+	[47] = "Avis de recherche !",	-- Wanted: Lieutenant Fangore
```

Comment:
`ObjectNames` are not manually translated but taken from [Wowhead](https://fr.wowhead.com/object=47/avis-de-recherche) so I tend to believe this is the symbol that the game uses.